src/main/java/org/broadinstitute/hellbender/tools/FlowBasedArgumentCollection.java

```diff
@@ -26,7 +26,7 @@ public class FlowBasedArgumentCollection implements Serializable {
 
 
 
-    public static final double DEFAULT_FILLING_VALUE = 0.001;
+    public static final double DEFAULT_FILLING_VALUE = 0.0001;
```

Comment:
Why the change? I think 0.001 is the right value (sorry if the comments were confusing)

Reply:
Value restored.